<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-0U1PUKJ" failures="0" tests="2" name="testCases.TC_001_AccountRegistrationTest2" time="0.001" errors="1" timestamp="2024-01-19T14:59:50 IST" skipped="1">
  <testcase classname="testCases.TC_001_AccountRegistrationTest2" name="setup" time="0.001">
    <error message="Can&#039;t find bundle for base name config, locale en_IN" type="java.util.MissingResourceException">
      <![CDATA[java.util.MissingResourceException: Can't find bundle for base name config, locale en_IN
at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2059)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1697)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1600)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1555)
at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:861)
at testBase.BaseClass.setup(BaseClass.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <system-out/>
  <testcase classname="testCases.TC_001_AccountRegistrationTest2" name="test_account_Registration" time="0.000">
    <skipped/>
  </testcase> <!-- test_account_Registration -->
  <system-out/>
</testsuite> <!-- testCases.TC_001_AccountRegistrationTest2 -->
